lib/libutils/ext/trace.c

```diff
@@ -53,12 +53,13 @@ int trace_get_level(void)
 
 static const char *trace_level_to_string(int level, bool level_ok)
 {
-	static const char lvl_strs[][4] = {
-		"UKN", "ERR", "INF", "DBG", "FLW" };
+	static const char lvl_strs[][9] = {
+		"UNKNOWN:", "ERROR:  ", "INFO:   ", "DEBUG:  ",
```

Comment:
Fair enough, I'd still like to have shorter meta data in the log lines. Perhaps with some flag to remove thread id, function name line number.